otherlibs/dune-glob/test/dune_glob_tests.ml

```diff
@@ -67,3 +67,12 @@ let%expect_test _ =
   [%expect {| [pass] "*" matches "foo.ml" == true |}];
   test glob "foo/" ~expect:false;
   [%expect {| [pass] "*" matches "foo/" == false |}]
+
+let%expect_test _ =
```

Comment:
It's a bit odd that we test the semantics of a particular string, rather than the semantics of `standard_dirs`. If we ever change the definition of `standard_dirs`, this test isn't going to save us.

Reply:
It's still a useful test to make sure we don't change the syntax of glob.

Here's a proper test for this feature though: https://github.com/ocaml/dune/pull/7808